src/main/features/dashboard/views/macro/claimStatus/claimPartAdmittedDefendant.njk

```diff
@@ -22,22 +23,36 @@
+
   <div class="status-block">
     <h2 class="heading-small">{{ t('You’ve responded to the claim') }}</h2>
-    <p>{% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    <p>{% if paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
         {{ t('You’ve said you owe {{ amount }} and offered to pay {{ claimantName }} immediately.',
-          { amount: claim.response.amount | numeral, claimantName: claim.claimData.claimant.name }) }}
-      {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
-        {{ t('You’ve said you owe {{ amount }} and offered to pay {{ claimantName }} on {{ paymentDate }}.',
-          { amount: claim.response.amount | numeral, claimantName: claim.claimData.claimant.name, paymentDate: claim.response.paymentIntention.paymentDate | date }) }}
-      {% elseif  claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
+          { amount: claim.response.amount | numeral, claimantName: claimant.name }) }}
+      {% elseif paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+        {{ t('You’ve said you owe {{ amount }} and offered to pay {{ claimantName }} by {{ paymentDate }}.',
+          { amount: claim.response.amount | numeral, claimantName: claimant.name,
+            paymentDate: paymentIntention.paymentDate | date }) }}
+      {% elseif paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
         {% set repaymentPlan = claim.response.paymentIntention.repaymentPlan %}
         {% set installmentAmount = repaymentPlan.instalmentAmount | numeral %}
         {% set paymentSchedule = t(repaymentPlan.paymentSchedule | renderPaymentSchedule) %}
-        {{ t('You’ve said you owe {{ amount }} and offered to pay {{ claimantName }} {{ installmentAmount }} {{ paymentSchedule }} starting {{ paymentDate }}.',
-          { amount: claim.response.amount | numeral, claimantName: claim.claimData.claimant.name,
-            installmentAmount: installmentAmount, paymentSchedule: paymentSchedule, paymentDate: repaymentPlan.firstPaymentDate | date }) }}
-      {% endif %}</p>
+        {{ t('You’ve said you owe {{ amount }} and offered to pay {{ claimantName }}
+            {{ installmentAmount }} {{ paymentSchedule }} starting {{ paymentDate }}.',
+          { amount: claim.response.amount | numeral, claimantName: claimant.name, installmentAmount: installmentAmount,
+            paymentSchedule: paymentSchedule | lower, paymentDate: repaymentPlan.firstPaymentDate | date }) }}
+      {% endif %}
+
+      {% if paymentIntention.paymentOption != domain.PaymentOption.IMMEDIATELY
+        and claim.claimData.defendant.isBusiness() %}
+        {{ t('You need to send them your company’s financial details if you haven’t already.') }}</p>
+        {{ internalLink('Get ' + claimant.name + '’s contact details.', claimantDetailsPageURI ) }}
+      {% endif %}
+    </p>
```

Comment:
Is this all in one giant <p>? please surround each {{ t() }} in it's own <p>

Reply:
Yes, there is that huge <P> because there is a sentence in the paragraph that is there depending if the defendant is business (and in this case there is an additional link as well)